cmd/tealdbg/cdtState.go

```diff
@@ -755,9 +893,27 @@ func makeTxnImpl(txn *transactions.Transaction, groupIndex int, preview bool) (d
 	return
 }
 
+func makeInnerTxnImpl(txn *transactions.SignedTxnWithAD, groupIndexes []int, preview bool) (desc []cdt.RuntimePropertyDescriptor) {
+	groupIndex := groupIndexes[len(groupIndexes)-1]
+	desc = makeTxnImpl(&txn.Txn, groupIndex, true, preview)
+
+	logs := makeArray("logs", len(txn.EvalDelta.Logs), encodeLogsID(groupIndexes))
+	innerTxns := makeArray("innerTxns", len(txn.EvalDelta.InnerTxns), encodeNestedInnerTxnID(groupIndexes))
+	createdApplicationID := fieldDesc{Name: "CreatedApplicationID", Value: strconv.Itoa(int(txn.ApplicationID)), Type: "number"}
```

Comment:
nit: it's unlikley to matter for these fields, but most other fields use the `bigint` type instead of `number` in case they exceed JavaScript's `Number.MAX_SAFE_INTEGER` (2^53 - 1).

In this same vein, you could do `strconv.FormatUint(uint64(txn.ApplicationID), 10)` instead of `strconv.Itoa(int(txn.ApplicationID))` to allow 32+ bits for these fields.

Reply:
Good point, will do